### ISSUE TITLE
Using with Express 4?

### ISSUE THREAD

**Issue by IngwiePhoenix** [jtblin/master-cluster#1]
You only specified a plain http example. Since the signature for an Express middleware is only one argument longer but overall the same as a typical http callback, I doubt it'd be hard to implement that.

Would be nice to know how this works, because it would help me a lot - especially the live-reloading caught me for good.

**Comment by jtblin** [comment]
Yes it's extremely simple to use it with express and adding it to the examples in the readme is long overdue. Here is how I use it with express, I'll add it to the readme later:

```
var app = require('express')();
var MC = require('master-cluster');
var responseTime = require('response-time');

// setup middlewares, mount routes, etc.
app.use(responseTime({ header: 'x-response-time' }));
app.get('/', function (req, res) {
  res.end('Hello world');
});
MC.createHttpServer(app, 3000);
console.log('Server listening on %d', 3000);
```

Hope this helps, glad that you like it.

**Comment by IngwiePhoenix** [comment]
That truely is dead simple. :) Thanks a lot!

**Comment by IngwiePhoenix** [comment]
I forgot to mention another thing!

I am running a security handler. So currently I am doing this:

``` javascript
...
var httpServer = app.listen(config.BIRD3.http_port, config.BIRD3.host);
io.listen(httpServer);
httpServer.on("listening", function(){
...
```

Now, how do I get to do this in MC? The thing is, that in one of these functions underneath, I am dropping my previleges from the current to a lower previleged user. Can I somehow listen to that event with MC?

**Comment by jtblin** [comment]
Not right now but it would be fairly simple, just need to return the http server in `createHttpServer`, see https://github.com/jtblin/master-cluster/blob/master/master-cluster.js#L67.

**Comment by IngwiePhoenix** [comment]
I see! Well, can you add it? :)

**Comment by jtblin** [comment]
Sure but not before a few days. PR welcome ;)

**Comment by jtblin** [comment]
Done, published as `0.1.1`.
